### PR TITLE
Acrolinx 1/31

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

**Comment by gewarren** [comment]
Close/reopen.

File: docs/core/tools/dependencies.md [hunk 1]
```diff
@@ -17,24 +17,24 @@ The `<PackageReference>` has the following basic structure:
...
+If you are familiar with MSBuild, it will look familiar to the other reference types that already exist. The key is the `Include` statement, which specifies the package ID that you wish to add to the project. The `<Version>` child element specifies the version to get. The versions are specified as per [NuGet version rules](/nuget/create-packages/dependency-versions#version-ranges).
 
 > [!NOTE]
-> If you are not familiar with the overall `csproj` syntax, see the [MSBuild project reference](/visualstudio/msbuild/msbuild-project-file-schema-reference) documentation for more information.
+> If you're not familiar with the project-file syntax, see the [MSBuild project reference](/visualstudio/msbuild/msbuild-project-file-schema-reference) documentation for more information.
 
-Adding a dependency that is available only in a specific target is done using conditions like in the following example:
+Use conditions to add a dependency that's available only in a specific target as shown in the following example:
```
Comment (tdykstra) [inline docs/core/tools/dependencies.md]:
```suggestion
Use conditions to add a dependency that's available only in a specific target, as shown in the following example:
```

File: docs/core/tools/dependencies.md [hunk 2]
```diff
@@ -17,24 +17,24 @@ The `<PackageReference>` has the following basic structure:
...
+The dependency will only be valid if the build is happening for that given target. The `$(TargetFramework)` in the condition is an MSBuild property that's being set in the project. For most common .NET Core applications, you will not need to do this.
 
 ## Add a dependency to the project
 
 Adding a dependency to your project is straightforward. Here is an example of how to add Json.NET version `9.0.1` to your project. Of course, it is applicable to any other NuGet dependency.
 
-When you open your project file, you will see two or more `<ItemGroup>` nodes. You will notice that one of the nodes already has `<PackageReference>` elements in it. You can add your new dependency to this node, or create a new one; it is up to you, as the result will be the same.
+When you open your project file, you will see two or more `<ItemGroup>` nodes. You will notice that one of the nodes already has `<PackageReference>` elements in it. You can add your new dependency to this node or create a new one; the result will be the same.
```
Comment (tdykstra) [inline docs/core/tools/dependencies.md]:
```suggestion
Your project file has two or more `<ItemGroup>` nodes. One of the nodes already has `<PackageReference>` elements in it. You can add your new dependency to this node or create a new one; the result will be the same.
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple suggestions for you to consider.

**Comment by gewarren** [comment]
@tdykstra Thanks for reviewing!

**Pull request merged**